virtcontainers/device/drivers/vfio.go

```diff
@@ -33,7 +33,9 @@ const (
 type VFIODevice struct {
 	DevType    config.DeviceType
 	DeviceInfo config.DeviceInfo
+	VFIOType   VFIODevice_Type
```

Comment:
you shouldn't use camel case :smile:

Reply:
Define the VFIODevice struct as below and move "VFIOType" "BDF" "SysfsDev" into "config.VFIODev"
Thanks for correcting me if any mis-understanding.

type VFIODevice struct {                                                            
    DevType    config.DeviceType                                                        
    DeviceInfo config.DeviceInfo                                                   
    VfioDevs    config.VFIODev                                                    
}